frontend/public/locales/en/distros.json

```diff
@@ -246,18 +246,6 @@
   "pop_os": {
     "distroName": "Pop!_OS",
     "introduction": "<0>Flatpak support is built into Pop!_OS 20.04 and newer—no setup required!</0> <p>If you are using an older version, you can refer to the instructions below.</p>",
```

Comment:
This was was removed as well:

"<p>If you are using an older version, you can refer to the instructions below.</p>"